pydantic_ai_slim/pydantic_ai/messages.py

```diff
@@ -117,17 +117,12 @@ def model_response(self) -> str:
 
 
 @dataclass
-class ModelTextResponse:
+class TextItem:
```

Comment:
This should probably be renamed. It might be possible to remove it entirely and just use `str` in its place, but then we would lose discriminators, so I think it's probably worth keeping this type.

Reply:
Decision: call things Parts